packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -449,6 +449,8 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly IOT_SITEWISE_DATA = new InterfaceVpcEndpointAwsService('iotsitewise.data');
   public static readonly IOT_TWINMAKER_API = new InterfaceVpcEndpointAwsService('iottwinmaker.api');
   public static readonly IOT_TWINMAKER_DATA = new InterfaceVpcEndpointAwsService('iottwinmaker.data');
+  public static readonly KAFKA = new InterfaceVpcEndpointAwsService('kafka');
```

Comment:
Checking the documentation for CN regions, can we add this as exceptions under China regions
https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-Ningxia.html
https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-Beijing.html

Reply:
In the `InterfaceVpcEndpointAwsService` class, the VPC endpoint path is generated as `${prefix || defaultEndpointPrefix}.${regionPrefix}${name}${defaultEndpointSuffix}`.

In this context, is it possible to confirm the `defaultEndpointPrefix`(`cn.com.amazonaws` or `com.amazonaws`) for the China region from the documentation?

I couldn't find the information in the provided link, and since I'm unable to run AWS CLI, I'm unsure how to confirm this.

```sh
❯ aws ec2 describe-vpc-endpoint-services --filters Name=service-type,Values=Interface Name=owner,Values=amazon --region ap-northeast-1 --query ServiceNames | grep kafka
    "com.amazonaws.ap-northeast-1.kafka",

❯ aws ec2 describe-vpc-endpoint-services --filters Name=service-type,Values=Interface Name=owner,Values=amazon --region cn-northwest-1 --query ServiceNames             

An error occurred (AuthFailure) when calling the DescribeVpcEndpointServices operation: AWS was not able to validate the provided access credentials
```

Reply:
i see in the linked documentation above it is mentioned as below: 

```
`Amazon Managed Streaming for Apache Kafka | kafka.cn-northwest-1.amazonaws.com.cn`
-- | --
```

```
Amazon Managed Streaming for Apache Kafka | kafka.cn-north-1.amazonaws.com.cn
-- | --
```

Reply:
From current implementation of `InterfaceVpcEndpointAwsService` class, we can get endpoint path for the China region like `cn.com.amazonaws.cn-north-1.{serviceName}.cn` and this is completely different from `{serviceName}.cn-north-1.amazonaws.com.cn` described in linked document.

Please let me know if there are any misunderstandings. I would appreciate it.

Reply:
okay, i think above link is not the VPC endpoints as usually VPC endpoints are generated in reverse order of suffix, the correct endpoint can be confirmed for China regions only with CLI command. For now, let's go with the current PR to add it as an endpoint with suffix `com.amazonaws` .